provider/ansible/facts.rb

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'provider/config'
+require 'provider/core'
+require 'provider/ansible/manifest'
+require 'provider/ansible/example'
+require 'provider/ansible/documentation'
+require 'provider/ansible/module'
+require 'provider/ansible/request'
+require 'provider/ansible/resourceref'
+require 'provider/ansible/resource_override'
+require 'provider/ansible/property_override'
+require 'provider/ansible/selflink'
+
+module Provider
```

Comment:
This doesn't seem to line up with the code below in `provider/config.rb`.  Am I misreading?